### PR TITLE
fix: added _options to topOptions parameters.context.google check

### PR DESCRIPTION
**Pull request by giorod3**
If _options is not present in the parameters.context.google object then line 78 breaks on getting params from undefined.

### CODE REVIEW

**Comment by giorod3** [comment]
I signed it!

File: src/apirequest.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ async function createAPIRequestAsync<T>(parameters: APIRequestParams) {
   // Create a new params object so it can no longer be modified from outside
   // code Also support global and per-client params, but allow them to be
   // overriden per-request
-  const topOptions = parameters.context.google
+  const topOptions = parameters.context.google._options
```
Comment (JustinBeckwith) [inline src/apirequest.ts]: Thank you!  Could you add a double layer of protection here?

```suggestion
  const topOptions = (parameters.context.google && parameters.context.google._options) 
```

Comment (giorod3) [reply src/apirequest.ts]: do you also want to validate params. as so: 
  const topOptions = (parameters.context.google && parameters.context.google._options && parameters.context.google._options.params)

Comment (JustinBeckwith) [reply src/apirequest.ts]: I don't think that's strictly required here.  To be honest I'm surprised there are situations where any of these are null/undefined, but it doesn't hurt to be careful :)

**Comment by giorod3** [comment]
done, thank you for the prompt response!

**Comment by bcoe** [comment]
@giorod3 I think you just need to run `npm run fix`, you appear to be having some TypeScript linting issues, hence the breaking builds.

**Comment by giorod3** [comment]
> @giorod3 I think you just need to run `npm run fix`, you appear to be having some TypeScript linting issues, hence the breaking builds.

@bcoe we are using this in js. I do not have the command `npm run fix`. On multiple machines and from multiple users the issue keeps arising. It also happens across all apis, not just one.

**Pull request merged**